src/ert/gui/simulation/experiment_panel.py

```diff
@@ -289,11 +288,20 @@ def toggleExperimentType(self) -> None:
             widget = self._experiment_widgets[self.get_current_experiment_type()]
             self._experiment_stack.setCurrentWidget(widget)
             self.validationStatusChanged()
+            if self.run_button.text() != EXPERIMENT_IS_RUNNING_BUTTON_MESSAGE:
+                if current_model.name() == "Manual update":
+                    self.run_button.setText(EXPERIMENT_IS_MANUAL_UPDATE_MESSAGE)
```

Comment:
I think regardless of `current_model.name()` I think we want

```
self.run_button.setTooltip("Execute Selected")
```

So that just the play symbol is shown, but on mouse hover you show "Execute selected"